lib/rules/media-feature-range-notation/README.md

```diff
@@ -84,3 +84,35 @@ The following patterns are _not_ considered problems:
 ```css
 @media (min-width: 1px) and (max-width: 2px) {}
 ```
+
+## Optional secondary options
+
+### `except`
+
+```json
+{ "except": ["array", "of", "options"] }
+```
+
+#### `"exact-value"`
+
+Ignore media features with exact values when the `context` option is selected.
```

Comment:
```suggestion
Reverse the primary option for media features with exact values.
```

To be consistent with our [`except` options](https://stylelint.io/developer-guide/rules#add-options) in other rules, let's have it reverse the primary option.

Let's add tests for:

```json
{
  "media-feature-range-notation": ["prefix", { "except": ["exact-value"] }]
}
```

which allows:

```css
@media (width = 1px) {}
```

```css
@media (min-width: 1px) {}
```

and disallows:

```css
@media (width > 1px) {}
```

```css
@media (width: 1px) {}
```

No need to add docs for this, as we typically only show one example of the primary option being reversed.